### PR TITLE
Soc:arm: Pandora STM32L475 Fix vendor from ST to Alientek

### PR DESCRIPTION
**Pull request by supperthomas**
https://github.com/zephyrproject-rtos/zephyr/issues/59183

### CODE REVIEW

**Review by kartben — DISMISSED** [review]

**Review by nashif — APPROVED** [review]

File: boards/arm/pandora_stm32l475/doc/index.rst [hunk 1]
```diff
@@ -1,6 +1,6 @@
 .. _stm32l475ve_pandora_board:
 
-ST STM32L475 Pandora
+Alientek STM32L475 Pandora
 ####################
```
Comment (nashif) [inline boards/arm/pandora_stm32l475/doc/index.rst:4]: need to make the ### longer

**Review by nashif — CHANGES_REQUESTED** [review]
lengthen the headline to and make the same length as the text.

**Review by kartben — DISMISSED** [review]

**Review by nordicjm — CHANGES_REQUESTED** [review]
Needs message text describing change

**Comment by supperthomas** [comment]
> Needs message text describing change

ok

**Review by erwango — CHANGES_REQUESTED** [review]
Thanks for taking care of this.

Please aslo fix board compatible:
`compatible = "st,pandora_stm32l475";` > `compatible = "alientek,pandora_stm32l475";`

Note that this might require to Add Alientek in `dts/bindings/vendor-prefixes.txt`

**Comment by supperthomas** [comment]
> Thanks for taking care of this.
> 
> Please aslo fix board compatible: `compatible = "st,pandora_stm32l475";` > `compatible = "alientek,pandora_stm32l475";`
> 
> Note that this might require to Add Alientek in `dts/bindings/vendor-prefixes.txt`

by the way. the MCU is STM32. the vendor just the little company to design the board. Do we need change the dts?

**Comment by erwango** [comment]
> > Thanks for taking care of this.
> > Please aslo fix board compatible: `compatible = "st,pandora_stm32l475";` > `compatible = "alientek,pandora_stm32l475";`
> > Note that this might require to Add Alientek in `dts/bindings/vendor-prefixes.txt`
> 
> by the way. the MCU is STM32. the vendor alientek is just the little company to design the board. Do we need change the dts? Actually, the MCU is ST vendor.

Yes, see `boards/arm/adafruit_feather_nrf52840/adafruit_feather_nrf52840.dts` for instance

**Comment by supperthomas** [comment]
> > > Thanks for taking care of this.
> > > Please aslo fix board compatible: `compatible = "st,pandora_stm32l475";` > `compatible = "alientek,pandora_stm32l475";`
> > > Note that this might require to Add Alientek in `dts/bindings/vendor-prefixes.txt`
> > 
> > by the way. the MCU is STM32. the vendor alientek is just the little company to design the board. Do we need change the dts? Actually, the MCU is ST vendor.
> 
> Yes, see `boards/arm/adafruit_feather_nrf52840/adafruit_feather_nrf52840.dts` for instance

ok

**Review by nordicjm — APPROVED** [review]

**Comment by nordicjm** [comment]
You need to sign this off again using your real name

**Comment by supperthomas** [comment]
Is it ok?

**Comment by supperthomas** [comment]
> > Is it ok?
> 
> Looked good. Don't hesitate to reopen. Also, there are a lot of PR and few reviewers (who have their own tasks as well), so answers may take some time to arrive. Thanks for you work anyway.

thank you. Very agree with you.
I change the name and PR
https://github.com/zephyrproject-rtos/zephyr/pull/59460

**Pull request closed without merging**